src/data/accessibility-quiz.ts

```diff
@@ -334,16 +343,32 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete"
   },
   {
-    Question:
-      "What is the minimum contrast ratio needed, according to the WCAG?",
+    Question: "What is the function of the `<contentinfo>` ARIA role?",
+    Answer: "displays content at that bottom of a website",
+    Distractor1: "displays content",
+    Distractor2: "displays a list",
+    Distractor3: "creates content",
```

Comment:
let's make these a little more wrong because technically it does define some sort of content. 
We just want to be more specific about what kind of content 

```suggestion
    Distractor1: "defines the navigation content",
    Distractor2: "displays a list",
    Distractor3: "defines article element content",
```